src/components/area-card/AreaCard.js

```diff
@@ -15,11 +20,16 @@ class AreaCard extends React.Component {
           <div className="area-detail">
             <div className="area-title">
               <figcaption className="text -small-title">{area.name}</figcaption>
-              { user.id === area.userId &&
-                <Link className="c-button -circle -transparent" to={`/areas/${area.id}`}>
-                  <Icon className="-small -gray" name="icon-edit"/>
-                </Link>
-              }
+              <div className="area-actions">
+                { user.id === area.userId &&
+                  <Link className="c-button -circle -transparent" to={`/areas/${area.id}`}>
+                    <Icon className="-small -gray" name="icon-edit"/>
+                  </Link>
+                }
+                <button className="c-button -circle -transparent" onClick={this.handleAreaDelete}>
```

Comment:
Does team areas appear in this page? If so you’ll need to check for user.id (like in edit), not all users should be able to delete all areas.

Reply:
True, I forgot that part :( THanks